src/components/SourcesTree.js

```diff
@@ -123,6 +129,11 @@ let SourcesTree = React.createClass({
     const { focusedItem, sourceTree, parentMap } = this.state;
     const isEmpty = sourceTree.contents.length === 0;
 
+    if (isEnabled("showSource")) {
+      const { shownSource } = this.props;
+      shownSource;
```

Comment:
I was wondering because the `shownSource` is a string, and then it's just invoked inline at 134, so it didn't seem to do much?